modules/Microsoft.Web/staticSites/deploy.bicep

```diff
@@ -65,7 +65,7 @@ param userAssignedIdentities object = {}
 @description('Optional. Specify the type of lock.')
 param lock string = ''
 
-@description('Optional. Configuration details for private endpoints. For security reasons, it is recommended to use private endpoints whenever possible.')
+@description('Optional. Configuration details for private endpoints.')
```

Comment:
Please keep the removed sentence for consistency with other modules

Reply:
I always use the field description generated in the API documentation with autorest from the official specs. Please add any CARML specific deviations to our docs here :)

Reply:
@eriqua + 1.

While we (most of the time) take the description from the API as well, we sometimes add further information like in this case. So the ask would be to just not remove the extra description :)

Reply:
I just blindly copied the text from the specs, this has slipped :D